tests/profile/admin/test_profile_highlights_admin.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from django.contrib.admin.sites import AdminSite
+from django.test import SimpleTestCase, TestCase
+
+from tests.profile.factories import ProfileHighlightFactory, ProfileFactory
+from tests.datasets.factories import DatasetFactory, IndicatorFactory
+from wazimap_ng.profile.models import ProfileHighlight
+from wazimap_ng.profile.admin.admins import ProfileHighlightAdmin
+
+class MockRequest:
+    pass
+
+
+class MockSuperUser:
+    def has_perm(self, perm, obj=None):
+        return True
+
+    @property
+    def is_superuser(self):
+        return True
+
+
+request = MockRequest()
+request.user = MockSuperUser()
+
+class ProfileHighlightAdminTests(TestCase):
```

Comment:
Could you please change this test to use pytest instead of xUnit? I would prefer if we were consistent